include/fmt/core.h

```diff
@@ -788,23 +795,25 @@ template <typename T> class buffer {
   auto end() const FMT_NOEXCEPT -> const T* { return ptr_ + size_; }
 
   /** Returns the size of this buffer. */
-  auto size() const FMT_NOEXCEPT -> size_t { return size_; }
+  FMT_CONSTEXPR20 auto size() const FMT_NOEXCEPT -> size_t { return size_; }
```

Comment:
Can this and some other simpler cases be constexpr instead of `FMT_CONSTEXPR20`?

Reply:
This would probably work for this line.
I was just placing `constexpr` or `FMT_CONSTEXPR` for classes only if these classes can be used at compile-time in C++11 or C++14 respectively. Looks like I had problems with making some of methods of `buffer` class `constexpr` in C++14 or lower, so I chose C++20.
I can just recheck which methods were causing problems here. Or, I can mark these simple methods `constexpr` despite class `constexpr`ness, but I don't see any advantages in this case.

Reply:
`constexpr` is slightly cleaner so I'd suggest using it when possible (i.e. for simple functions).

Reply:
`constexpr` applied where it's possible, but of course, I could miss something